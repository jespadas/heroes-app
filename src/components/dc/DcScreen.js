import React from 'react';
import { HeroList } from '../heroes/HeroList';

export const DcScreen = () => {

    const DcHeroes = 'DC Comics';

    return (
        <div>
            <h1>DcScreen</h1>
            <hr />
            <HeroList publisher={DcHeroes} />
        </div>
    )
}
