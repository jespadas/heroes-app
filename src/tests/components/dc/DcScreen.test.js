import { shallow } from "enzyme";
import { DcScreen } from "../../../components/dc/DcScreen";

describe('Test in <DcScreen />', () => {

    const wrapper = shallow(<DcScreen />);

    test('Should be displayed correctly', () => {

        expect(wrapper).toMatchSnapshot();

    });

});
