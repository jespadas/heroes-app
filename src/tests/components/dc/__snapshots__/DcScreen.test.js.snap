// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <DcScreen /> Should be displayed correctly 1`] = `
<div>
  <h1>
    DcScreen
  </h1>
  <hr />
  <HeroList
    publisher="DC Comics"
  />
</div>
`;
